src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/PosteriorProbabilitiesUtils.java

```diff
@@ -134,9 +135,16 @@ else if (a.isSymbolic()) {   //use the greater of the priors for non-ref
             builder.phased(vc1.getGenotype(genoIdx).isPhased());
             if ( posteriors.get(genoIdx) != null ) {
                 GATKVariantContextUtils.makeGenotypeCall(vc1.getMaxPloidy(HomoSapiensConstants.DEFAULT_PLOIDY), builder,
-                        GenotypeAssignmentMethod.USE_PLS_TO_ASSIGN, posteriors.get(genoIdx), vc1.getAlleles(), null);
+                        GenotypeAssignmentMethod.USE_PLS_TO_ASSIGN, posteriors.get(genoIdx), vc1.getAlleles(), vc1.getGenotype(genoIdx), null);
+
                 builder.attribute(GATKVCFConstants.PHRED_SCALED_POSTERIORS_KEY,
-                        Utils.listFromPrimitives(GenotypeLikelihoods.fromLog10Likelihoods(posteriors.get(genoIdx)).getAsPLs()));
+                            Utils.listFromPrimitives(GenotypeLikelihoods.fromLog10Likelihoods(posteriors.get(genoIdx)).getAsPLs()));
+                //TODO: refactor me -- this is to make up for the GGVCFs changes fixing GQ0 hom-refs
+                if (!builder.makeWithShallowCopy().hasPL()) {
```

Comment:
Did you check using IntelliJ?  It has a fairly generic name, but it's just in CalculateGenotypePosteriors and a HaplotypeCaller GVCF feature I tried and ought to deprecate because it actually made for bigger GVCFs.
<img width="485" alt="image" src="https://github.com/broadinstitute/gatk/assets/6578548/66714992-9135-4dd1-b680-61ebe18944a0">
I don't believe M2 touches this at all. You're probably thinking of the posteriors in the AlleleFrequency Calculator

Reply:
(Those numbers admittedly don't d add up, but I'm not hiding anything. The other hits are in the PosteriorProbabilityUtilsUnitTest)
<img width="488" alt="image" src="https://github.com/broadinstitute/gatk/assets/6578548/a3bfc309-a0c9-447f-be88-f5a4eba43d1d">